pkg/mkimage-raw-efi/install

```diff
@@ -168,12 +168,20 @@ zfs_umount() {
   umount /root/dev ||:
 }
 
+
+cmdline="/proc/cmdline"
 logmsg "EVE-OS installation started"
+if grep -q interactive /proc/cmdline; then
+  logmsg "Interactive mode"
+  bash ./parse-interactive-installer-conf
+  cmdline=/tmp/newcmdline
```

Comment:
Why use a file for the output?
You can do newcmdline=$(./parse-interactive-installer-conf.sh)
and then check if that string is empty

Reply:
Ah - your name confused me (hence a bunch of comments below).
Had you named the variable cmdlinefile I would have not been confused.

Reply:
I renamed the variable to cmdlinefile.